ddtrace/tracer/tracer_test.go

```diff
@@ -109,10 +109,10 @@ func TestTracerCleanStop(t *testing.T) {
 		defer wg.Done()
 		for i := 0; i < n; i++ {
 			// Lambda mode is used to avoid the startup cost associated with agent discovery.
-			Start(withTransport(transport), WithLambdaMode(true))
+			Start(withTransport(transport), WithLambdaMode(true), withNoopStats())
 			time.Sleep(time.Millisecond)
-			Start(withTransport(transport), WithLambdaMode(true), WithSampler(NewRateSampler(0.99)))
-			Start(withTransport(transport), WithLambdaMode(true), WithSampler(NewRateSampler(0.99)))
+			Start(withTransport(transport), WithLambdaMode(true), WithSampler(NewRateSampler(0.99)), withNoopStats())
+			Start(withTransport(transport), WithLambdaMode(true), WithSampler(NewRateSampler(0.99)), withNoopStats())
```

Comment:
Since this change was also submitted in https://github.com/DataDog/dd-trace-go/pull/1052 I'm wondering if it's supposed to be part of this PR or got in here accidentally.

Reply:
You're right, it's not part of this PR. #1052 has been directly incorporated for demonstration purposes (I was testing total time savings). When that is merged, I will resolve the conflicts here. For the time being, just ignore those changes here. 

If it's really troublesome, I can remove those changes from this PR.